lib/classes/eik-config.js

```diff
@@ -43,11 +43,14 @@ module.exports = class EikConfig {
         this[_tokens] = new Map(tokens);
 
         assert(
+            // @ts-ignore
```

Comment:
Super if you can do a search and replace `ts-ignore` with `ts-expect-error` so we get notified when they can be removed 🙏